### DISCUSSION TITLE
CC War mode?

### DISCUSSION THREAD

**Discussion by UndoUreche** [chromiecraft/chromiecraft#7206] [Suggestions]
![image](https://github.com/user-attachments/assets/5181ad2f-0dfe-49ba-a2bb-e195638a70d1)
> source https://wowpedia.fandom.com/wiki/War_Mode

My suggestion would be to replicate this retail feature on CC. Except for the pvp talents and world events, the latter of which only exist to motivate people to fight, a private server can provide some multipliers to xp, gold, honor or something else to make it worth it.

I believe the motivation should be something relevant to max level players, as it's more important for those to be brawling rather than ganking low level players. And that's it. Allow us to flag ourselves only in capitals, give some multipliers to flagged players in the open world and let them that decide to take the risk and enjoy the benefits.

Here are some benign ideas for potential rewards for turning it on:
- 10-15% damage to mobs as a buff
- 10-20% xp increase (this one has to be higher because of the potential for ganking)
- a 10% chance to get an extra item drop from mobs (probably too hard to implement)
 
 The point would be to make these rewards/multipliers impactful or visible enough that a player feels them, but also not enough to make people think that they're missing out if they don't partake.

**Comment by TotallyNotMyAccount** [comment]
I like it a lot. XP buff is probably still too small, I think turtle has 35% or something.

**Comment by krutoj87** [comment]
I don't like the idea. CC is a pve server and should stay that way. I don't think players should be worse off, because they decide to not participate in open world pvp on a pve server.

If the rewards are just visual things like titles or mounts for leveling with such a mode I would be fine with it. But it shouldn't be anything that actually enhances the damage or received exp or items. Players who choose to not participate in this mode shouldn't be worse off, than people who do.

**Reply by UndoUreche** [reply]
I agree with you, and that's precisely my point in the last paragraph.

Right now there's no benefit to partaking in world pvp. I'm suggesting we rectify that, by giving those that do partake a small increase in something (further details to be discussed) so as to make it worth removing the safety that not being pvp flagged provides.

The reward should be visible but it should only be enough to contend with the ability to farm peacefully with no pvp.

**Comment by SignorSimone** [comment]
The exp has already increased in the battlegrounds but there is still no participation. PvP is a state of mind, it is useless to convince those who do PvE, not even if you give them all the raid gear they will duel.
Personally I think it's useless, from every point of view, to ask GMs and devs to do something for PvP, the only weapon is to unite us PvPers and entice new people to come, because, the only thing we can do is bring new people with a vision more similar to ours.
Here they think like this, there is little to do except look for more open-minded people. Chromie rewards pvp player but say "Here do PvE, PvP its wrong!", we need to talk about the server and make it known.

**Reply by krutoj87** [reply]
A warmode would basically try to pressure people into playing in a way they don't like. In order to achieve this the benefits of the warmode would need to be massive. But if you do it this way PVE players would be put into a lose/lose situation. Either play in a way you dislike or accept major drawbacks compared to other players. That's why I'm against a warmode. In my opinion finding like minded players to join the server is the better option than pressuring PVE players to do PVP.

Besides getting the word out to make more PVP players join it might help to make some entry pvp sets more easily available. Because as I understood from world chat the state right now is, that you will be farmed in BGs until you have enough honor to buy pvp gear. That might be off putting for some. 

But I am sure there are still steps that can be taken without resorting to pressuring PVE players to play with PVP turned on. 

Edit:
Also to be more than just a victim in open world pvp you would need to farm pvp sets aswell. The warmode would just be a pleasant experience for pvp players but only has downsides for pve players.

**Reply by UndoUreche** [reply]
You've gotten part of it. 

Firstly I think you've got the wrong perspective. We're talking something along the lines of a 10% buff to damage against monsters (it's one of the suggestions). Is not having this a major drawback? Not being disturbed while farming is a massive benefit, to defeat that as an incentive you'd have to provide an equally massive reward and that's not what I'm arguing for. I'm arguing for a contending offer.

As for the edit. Yes. This is literally the point of warmode. To give people a reason to forsake safety and the motivation to prepare themselves for it by engaging with pvp content. Right now, if I flag myself I'll just farm some mobs, see a random unflagged player follow me around on a mount until I have 20% hp and get jumped on. Why even bother?

The point was NEVER to force anyone into playing flagged. The point was to encourage people to partake. I'm well aware that it's a matter incentive, it's precisely why I labeled my suggestions as "some benign ideas". I wanted them discussed and was hoping that we'd find some sort of reward that wouldn't be too enticing so as to force people, but also not meaningless enough that it won't make it worth it.

**Reply by krutoj87** [reply]
It doesn't really matter if you call it to motivate players or to pressure players, the idea is clear. You want to make players play in a way they font want to by dangling some shiny thing in front of them. In order for this to work you need to have some massive benefits, else people just won't do it. And if you make benefits big enough for making people play in a way they don't want to play you just make their overall experience worse because they are forced to make a lose/lose decision. You basically increase your fun on the cost of their fun. 

Let's face it the warmode is a win/win for pvp players, because you either get to do open world pvp or in the worst case you just play the game as you do already just with some additional benefits. 
For pve players on the other hand it's lose/lose. Either you play in a way you don't want to play in, or you lose out on benefits others will get. 

You call it incentive, but the point of this is really to push people into something they don't want. So there is no middle ground where this will be ok for pve players but also makes them do something they don't want to do. 

I'm still against it. Just let pve players play without open world pvp on a pve server. If you want to make more open world pvp happen on the server try to bring more likeminded players to the server instead of making the game experience worse for the people who don't want to have anything to do with pvp.

**Reply by UndoUreche** [reply]
Of course there's a middle ground. Going through the plaguelands, hellfire, or whatever other zone provides some tower buff and not having it doesn't compel me to capture towers because I don't care enough to do it.
Conversely you can get quite significant buffs for pve content by bringing your guild mandated consumes, hanging on to neck buffs and so on and most people do not do it. Are they missing out? Do they feel forced? Hardly.

I'd say you're overreacting if you're trying to argue that any buff no matter how small would force anybody into a particular gamemode. And please understand that nobody is forcing anything on anyone. This should be as simple as choosing to toggle transmog. I don't see how having a reward for the **optional** risks you willingly choose to take as a player somehow degrades the experience of other players.

**Reply by TotallyNotMyAccount** [reply]
You say the solution is to attract more PvP players to the realm instead of Warmode. How are we supposed to do that when BGs pop rarely, arena is dead and the server doesn't even have open world PvP? If we want more active pvp we need to incentivise it. How is warmode detrimental to you? Everything stays as it is now, hell maybe some open world ressources even get cheaper because people are more easily able to farm them. Please explain how other people getting a buff impacts your gold/h or fun. This is such a bucket of crabs mentality.

**Reply by krutoj87** [reply]
Do I really need to explain how it impacts my fun if others have benefits I don't have?
With your reasoning we could also add p2w by making increased exp rates or item drops buyable with Chromiepoints to give people an incentive to support CC. I'd even be more ok with this, than with locking benefits behind playing with pvp mode.

Also it's not on me to find solutions for your problem. Most of the population is fine with the way it is and me too. I am totally fine with mit changing anything. If the current situation isn't good enough for you, than think of ways of bringing more like minded players to the server or just accept the fact, that it's a pve server and open world pvp won't be a thing. 
But a solution that makes the game miserable for most other players isn't a solution that even deserves to be discussed. 

You chose to play on a pve server. You knew there wouldn't be open world pvp when you joined. There are plenty of pvp servers out there and theres even a progressive one with Warmane Onyxia.
But this is the only progressive pve server with a decent population. And the reason I joined this server is because I detest open world pvp and don't want to have anything to do with it. So I of course don't want to be pushed into playing with pvp turned on and I also don't want others to have advantages I don't have.

**Reply by UndoUreche** [reply]
Well forgive me for pointing it out like this, but this is a very selfish point of view, and it comes with quite a few flaws.

> With your reasoning we could also add p2w by making increased exp rates or item drops buyable with Chromiepoints to give people an incentive to support CC. 

No, with my reasoning the current incentives are perfectly fine. I keep saying it's a matter of execution. Incentives exist and they are fine.

> I'd even be more ok with this, than with locking benefits behind playing with pvp mode.

This is not a good hill to die on.

>  But a solution that makes the game miserable for most other players isn't a solution that even deserves to be discussed.

Your rationale for this seems to be "they get something I **choose** not to pursue so F them" which imo is a poor stance given that this suggestion pushes the change as a voluntary choice. I.e. your life won't get any more miserable than it is already, if this gets passed.

> You knew there wouldn't be open world pvp when you joined.

You may be surprised by this but no. I joined for the progressive nature of the server. And part of why I'm staying (beside sunk costs) is the open nature of the devs when it comes to change.

> So I of course don't want to be pushed into playing with pvp turned on and I also don't want others to have advantages I don't have.

Who would push you? A minor buff? Can't you live without it, just like you've been living without it for the past year or so of bc? Again, this is meant to be a choice.

As a closer, thank you for your perspective. I think it's best that we agree to disagree on this topic and move on rather than spin in circles all day.

**Comment by krutoj87** [comment]
All the excuses you make can also be made for p2w additions like increased exp and drop rates you can buy for money. How does it affect you if others buy them? It's optional you **choose** not to buy them. 

All you do here is argue semantics because incentive doesn't sound as bad. But semantics don't change the nature of what you are suggesting. What you suggest is to implement something that will pressure people to play pvp when they don't want to. 
Who is open to pvp plays pvp already. Who doesn't want to play pvp doesn't do it and you should respect that. 
Because your "incentives" only have the goal to trigger FOMO in pve players so they start doing pvp. You say we just need to find the right incentives, which are big enough to make people play pvp and don't bother pve players of they miss out on them. But that's just nonsense. If the lack of those "incentives" won't hurt the pve players you will achieve exactly nothing but wasting the time of the devs. Because people will just ignore them and not play pvp, because they don't want to play pvp. If the incentives are big enough to trigger the FOMO and pressure people into doing something they don't want to, it will just make the overall experience for those people worse, because they now need to do something they don't want to, just to have the same chances as others. And people who still refuse to play pvp will be effectively second class players because they level slower and get fewer items than others. 

As I wrote before your warmode is essentially a win/win for pvp players. That's why you trying so hard to defend it. It literally doesn't have a single downside for you. Worst case for you still won't get to do more pvp but at least you level faster and get more items. And in the best case you level faster, get more items and get your open world pvp on top. 
For pve players it's a lose/lose situation. That's why I'm absolutely against it. There is no benefit for pve players and only downsides. Because if your "incentives" force us to play how we don't want to play, then we are better off without those "incentives" at all. 
The decision I get to make as a pve player with this warmode is: To play how I hate to play just to level as fast and get as many items as others, or to just accept, that I'm a second class player who will level slower and get fewer items then the others. 

Also the reasons why you chose this server aren't the point. The point is, that you knew what type of server it is when you signed up for it. But now you try to change the type of this server into something nobody of us has signed up for. For me this warmode would ruin the server and I would be very pissed because all the time I've already put into my chars would have been wasted. So I hope the devs shut this down very fast. 

I said everything there is to say about this topic. This makes the experience worse for pve players, which is the majority of the playerbase on the server, without giving them any real benefit, that is more than just mitigating the downsides this feature created in the first place. As a pve player I would need to suddenly play with pvp turned on to get back to where I am currently already am. On the same level with equal chances as everybody else. 
I have sympathy for the pvp players on the server and wish for you to get more chances to play pvp. But not on the cost of the game experience of me and the other pve players. 

I'm going to deactivate notifications for this suggestion, because there won't be any new insights here. All this discussion is right now is discussing semantics and using soft language to make this suggestion not sound as bad as it is. There will be nothing that changes my mind. And I will never accept a change that requires me to play with pvp turned on, on a pve server, just to have equal chances as everyone else.

**Reply by TotallyNotMyAccount** [reply]
No, the person using disingenuous language here is very much you yourself, trying to turn a molehill into a mountain. Equating Warmode to P2W, really? To be frank, you are incredible hostile in general and are attacking people personally, by suggesting they have some nefarious scheme cooking to hurt the PvE community on this server. You do these things because you realize that your actual arguments are paper thin.

I get that you don’t engage in PvP and thus have little idea about the dynamics, but duking it out with another player for an hour is very bad XP and/or gold. It is terrible honor as well. This is to say nothing of having to switch farm spots or even zones if you get the short end of the stick. Then there’s stuff like making sure you never pull extra, or never go below 50% hp if opposing faction is around, both of which majorly impact your efficiency. The goal of Warmode is not to pressure PvE players into flagging themselves, it is to partially recompense those interested for the fact that the activity they find fun is not rewarded by the game at all otherwise.

We do not aim to get PvE players into WM, but to remove the FOMO already experienced every time by PvPers when they flag themselves as well as maybe attracting new players to the server. Furthermore, WM is obviously not something we came up with, but a fair solution to the problem, that already exists on other private servers, like Turtle, and was of course originally introduced in retail WoW. In none of these places did WMs introduction kill the PvE side of things, and speaking of Chromie specifically, anything that gets fresh players in is a good thing. 
This server is getting so small it’s tethering on the edge of viability. And then you get in here and go “This change doesn’t directly benefit me!” and have an absolute fit over it. I’m very negatively reminded of the discussion about mythic modes, where people even stood to benefit from tuning more appropriate to them, but went crazy over the possibility others might get any benefits out of playing in a way they themselves had no interest in.

**Reply by UndoUreche** [reply]
From this PVE only perspective (what I'm understanding is your perspective) we have had a significantly worse situation in the lv60 bracket. Back when AV rep was required for a 2h mace, and rings, or back when rank14 weapons became available. Back then, PVP content was tied to items and therefore directly tied to performance, and it did indeed lead to participation. Not only that but there wasn't half as much drama about it on the server as there is in this thread.

That trip down memory lane aside, I'm noticing that your bar for incentive is quite high. After everything I've read, I imagine you would be a person that (should this ever be adopted) would think that having the possibility fighting other players in the open world is too much of a downside when compared to whatever rewards it would provide and choose not to participate. This is precisely what I'm counting on: that people would judge for themselves the benefits against the massive downside of fighting others and at least some would join in what is effectively an alternative open world experience.

I find it insultingly dishonest of you when I see statements like "It literally doesn't have a single downside for you." and I hope you're kidding. Just as you've pointed out in your many complaints having to fight other players is by no means a benefit. This is exactly why it should be offset by something.
